src/puppetlabs/puppetdb/query/event_counts.clj

```diff
@@ -124,7 +124,10 @@
   "Convert an event-counts `query` and a value to `summarize_by` into a SQL string.
   A second `counts-filter` query may be provided to further reduce the results, and
   the value to `count_by` may also be specified (defaults to `resource`)."
-  ([version query [summarize_by {:keys [counts_filter count_by] :as query-options} paging-options]]
+  ([caller
```

Comment:
I'm not sure about this `caller` business. Would much rather call out what it changes explicitly. `include-latest-events`, perhaps, based on my reading.